src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -85,6 +85,44 @@ namespace ZoneAirLoopEquipmentManager {
 
     using namespace DataDefineEquip;
 
+    // constexpr std::array<std::string_view, static_cast<int>(ZnAirLoopEquipType::Num)> ZnAirLoopEquipTypeNames = {
+    //     "AirTerminal:DualDuct:ConstantVolume",
+    //     "AirTerminal:DualDuct:VAV",
+    //     "AirTerminal:SingleDuct:VAV:Reheat",
+    //     "AirTerminal:SingleDuct:VAV:NoReheat",
+    //     "AirTerminal:SingleDuct:ConstantVolume:Reheat",
+    //     "AirTerminal:SingleDuct:ConstantVolume:NoReheat",
+    //     "AirTerminal:SingleDuct:SeriesPIU:Reheat",
+    //     "AirTerminal:SingleDuct:ParallelPIU:Reheat",
+    //     "AirTerminal:SingleDuct:ConstantVolume:FourPipeInduction",
+    //     "AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan",
+    //     "AirTerminal:SingleDuct:VAV:HeatAndCool:Reheat",
+    //     "AirTerminal:SingleDuct:VAV:HeatAndCool:NoReheat",
+    //     "AirTerminal:SingleDuct:ConstantVolume:CooledBeam",
+    //     "AirTerminal:DualDuct:VAV:OutdoorAir",
+    //     "AirTerminal:SingleDuct:UserDefined",
+    //     "AirTerminal:SingleDuct:Mixer",
+    //     "AirTerminal:SingleDuct:ConstantVolume:FourPipeBeam"};
+
+    constexpr std::array<std::string_view, static_cast<int>(ZnAirLoopEquipType::Num)> ZnAirLoopEquipTypeNamesUC = {
+        "AIRTERMINAL:DUALDUCT:CONSTANTVOLUME",
+        "AIRTERMINAL:DUALDUCT:VAV",
+        "AIRTERMINAL:SINGLEDUCT:VAV:REHEAT",
+        "AIRTERMINAL:SINGLEDUCT:VAV:NOREHEAT",
+        "AIRTERMINAL:SINGLEDUCT:CONSTANTVOLUME:REHEAT",
+        "AIRTERMINAL:SINGLEDUCT:CONSTANTVOLUME:NOREHEAT",
+        "AIRTERMINAL:SINGLEDUCT:SERIESPIU:REHEAT",
+        "AIRTERMINAL:SINGLEDUCT:PARALLELPIU:REHEAT",
+        "AIRTERMINAL:SINGLEDUCT:CONSTANTVOLUME:FOURPIPEINDUCTION",
+        "AIRTERMINAL:SINGLEDUCT:VAV:REHEAT:VARIABLESPEEDFAN",
+        "AIRTERMINAL:SINGLEDUCT:VAV:HEATANDCOOL:REHEAT",
+        "AIRTERMINAL:SINGLEDUCT:VAV:HEATANDCOOL:NOREHEAT",
+        "AIRTERMINAL:SINGLEDUCT:CONSTANTVOLUME:COOLEDBEAM",
+        "AIRTERMINAL:DUALDUCT:VAV:OUTDOORAIR",
+        "AIRTERMINAL:SINGLEDUCT:USERDEFINED",
+        "AIRTERMINAL:SINGLEDUCT:MIXER",
+        "AIRTERMINAL:SINGLEDUCT:CONSTANTVOLUME:FOURPIPEBEAM"};
+
```

Comment:
Adds air terminal equipment name string_view array definition for use with Enum Class ZnAirLoopEquipType.